providers/aws/models.py

```diff
@@ -19,6 +19,15 @@ class AWS_Assume_Role:
     external_id: str
 
 
+@dataclass
+class Organizations_Info:
```

Comment:
I prefer to name it as the other classes with the `AWS` prefix. What do you think?